apps/api-reference/resource-miscellaneous.md

```diff
@@ -5,7 +5,7 @@ title: Miscellaneous
 ---
 <ul id="resource-list">
   {% for page in site.pages %}
-  {% assign misc = "currencies locales search start-page watched-products product-category-assignments" | split: ' ' %}
+  {% assign misc = "currencies locales search start-page watched-products product-category-assignment tax-classes" | split: ' ' %}
```

Comment:
you changed product-category-assignment**s** to product-category-assignment, sure?